run.sh

```diff
@@ -1,14 +1,22 @@
 #!/bin/bash
-###############################################################################
-# Build docker full image and run ROS code interactively for debugging        #
-###############################################################################
+######################################################################################
```

Comment:
Ideally in general we should stick to 80 chars (thanks punchcards) for readability, specially for line blockers like these. Multi-line comments are not so bad with automatic wrapping, but these ### are not so great.
I've been thinking whether to set a pre-commit check for this, what do you think?

Reply:
Pre-commit for ensuring this is a good idea. We can also run it locally for reference.

Reply:
I have reduced the comment section and line length by less that 80 characters.  Also,  changing the `###` for `---` to improve readability